tests/test_finder.py

```diff
@@ -975,5 +975,41 @@ def test_hybrid_multiconfig(self):
         # verify dumped files count and names (+1 as we match with template)
         dumps = self._verify_and_get_dumps(9, multistep=True)
 
+class CVParameterTest(unittest.TestCase):
```

Comment:
We already cover most of the default CV parameter functionality through the regular unit tests but apparently this was not covered in order to miss the encountered problem so +1.